docs/extras/building_vtk.rst

```diff
@@ -168,18 +173,20 @@ This script can be called with:
    dependencies.
 
 
-Python VTK Wheel on Raspberry PI (64-bit)
+Python VTK Wheel on Raspberry Pi (64-bit)
```

Comment:
One thing about this is not actually clear to me: the below docker workflow would be needed to build wheels on a PC, but not on the RPi itself, right? If this is the case, perhaps changing the title from "on Raspberry Pi" to "for Raspberry Pi" might help clear this without further explanations.

Reply:
> Or should the docker container be run on the RPi?

Depends.  Building a non Python source Linux wheel from scratch means compiling C libraries, which are linked to the version of glibc of the OS they were compiled on.  In practice in the past, this mean actually having a physical machine with an older version of Linux (e.g. CentOS 5) to compile wheels that were compatible with CentOS 5+.

Now through the magic of docker, you can build a wheel on your machine, use `auditwheel` to verify compiled binaries of the wheel, and have it tagged.

Reply:
Sorry, I'm not sure I understand the answer :) But if the docker things in this section are meant to be run on the RPi itself then it's all good.